apps/desktop-wallet/src/api/apiUtils.ts

```diff
@@ -16,8 +16,7 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { NFTTokenUriMetaData } from '@alephium/web3'
-import { FungibleTokenMetadata } from '@alephium/web3/dist/src/api/api-explorer'
```

Comment:
Unless I import `explorer`, the current bundling configuration cannot resolve these types. The web3 package would probably need to be updated for it to work, but I don't want to deal with it. Importing `explorer` works, so I'd rather roll with it.